data/pr_339.sql

```diff
@@ -0,0 +1,7 @@
+-- a drs_object's size can be really big
```

Comment:
Might want to change the comment here
```suggestion
-- add the statistics column if it doesn't already exist
```